include/gauge_field_order.h

```diff
@@ -905,17 +905,22 @@ namespace quda {
        deploy for a specifc field ordering, the two operator()
        accessors have to be specialized for that ordering.
 
-       @tparam Float Underlying type returned by the accessors
+       @tparam Float_ Underlying type returned by the accessors
        @tparam nColor Number of colors for the field
        @tparam nSpinCoarse Number of "spin degrees of freedom" (for coarse-link fields only)
        @tparam order Storage order of the field
        @tparam native_ghost Whether to use native ghosts (inlined into
+       @tparam storeFloat_ Underlying storage type for the field
        the padded area for internal-order fields or use a separate array if false)
      */
-    template <typename Float, int nColor, int nSpinCoarse, QudaGaugeFieldOrder order, bool native_ghost = true,
-              typename storeFloat = Float>
+    template <typename Float_, int nColor, int nSpinCoarse, QudaGaugeFieldOrder order, bool native_ghost = true,
+              typename storeFloat_ = Float_>
     struct FieldOrder {
 
+      /** Convenient types */
+      using Float = Float_;
+      using storeFloat = storeFloat_;
+
```

Comment:
Some noise here: in the MMA PR I used `type` and `store_type` [here](https://github.com/lattice/quda/blob/e3612009273c0c3a947f67b6faca675551edc3e4/include/gauge_field_order.h#L241). We should probably unify the namings at some point.

Reply:
...and I am now favouring `store_t` 😜

Reply:
So `store_t` and `compute_t`?

Reply:
I think that would be fine.  But please no global edits until generic kernels is merged in.  One global edit at a time is already one too many....

Reply:
...I'm going to wait to change this until we/you make a global decision in a later PR.

Reply:
Agreed!